script/update_beats.sh

```diff
@@ -22,19 +22,27 @@ git clone https://github.com/elastic/beats.git ${GIT_CLONE}
 
 # sync
 rsync -crpv --delete \
-    --exclude=.gitignore \
     --exclude=dev-tools/packer/readme.md.j2 \
-    --include="script/***" \
     --include="dev-tools/***" \
-    --include="libbeat/scripts/***" \
-    --include="libbeat/_meta/***" \
+    --include="script/***" \
+    --include="testing/***" \
+    --include="libbeat/" \
     --include=libbeat/Makefile \
+    --include="libbeat/_meta/***" \
+    --exclude="libbeat/_meta/fields.generated.yml" \
```

Comment:
@simitt Haven't tested it but was wondering if this could cause issues as I think the generated fields.yml from libbeat is used to generate the full fields.yml for apm-server.

Reply:
Thanks for pointing that out @ruflin I'll look into it and test it again. 
This file was never added to apm-server though - not in the original version before PR #449 , and not before this PR. There was an issue with the current `rsync` cmd and after fixing it, this file needed to be specifically excluded to reach the same state as before the change.

Reply:
@simitt Great, thanks for checking.